spec/base_spec_helper.rb

```diff
@@ -7,6 +7,7 @@
 require 'simplecov' if ENV["COVERAGE"]
 require 'rubygems'
 require 'pry' unless ENV['CI']
+require 'view_component/test_helpers'
```

Comment:
The tests don't work without this? :thinking:

Reply:
I could not figure out why `order_print_ticket_spec.rb` was failing after switching to the `system_spec` helper. By comparing feature and system spec I've found this one bit to be missing, and it seems to bring it to green, but I can't explain it either... Without this line it fails with this error:

`rspec ./spec/features/admin/order_print_ticket_spec.rb:36 # `


```
    As an administrator
    I want to print a ticket for an order
 as an enterprise manager viewing the edit page can print an order's ticket
     Failure/Error: print_data = page.evaluate_script('printData');
     
     Ferrum::JavaScriptError:
       ReferenceError: printData is not defined
           at <anonymous>:1:15

```